resources/s3-access-points.go

```diff
@@ -0,0 +1,80 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3control"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+func init() {
+	register("S3AccessPoint", ListS3AccessPoints)
+}
+
+type S3AccessPoint struct {
+	svc         *s3control.S3Control
+	accountId   *string
+	accessPoint *s3control.AccessPoint
+}
+
+func ListS3AccessPoints(s *session.Session) ([]Resource, error) {
+	// Lookup current account ID
+	stsSvc := sts.New(s)
+	callerID, err := stsSvc.GetCallerIdentity(&sts.GetCallerIdentityInput{})
+	if err != nil {
+		return nil, err
+	}
+	accountId := callerID.Account
+
+	resources := []Resource{}
+	svc := s3control.New(s)
+	for {
+		params := &s3control.ListAccessPointsInput{
+			AccountId: accountId,
+		}
+
+		resp, err := svc.ListAccessPoints(params)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, accessPoint := range resp.AccessPointList {
+			resources = append(resources, &S3AccessPoint{
+				svc:         svc,
+				accountId:   accountId,
+				accessPoint: accessPoint,
+			})
+		}
+
+		if resp.NextToken == nil {
+			break
+		}
+		params.NextToken = resp.NextToken
+	}
+
+	return resources, nil
+}
+
+func (e *S3AccessPoint) Remove() error {
+	_, err := e.svc.DeleteAccessPoint(&s3control.DeleteAccessPointInput{
+		AccountId: e.accountId,
+		Name:      aws.String(*e.accessPoint.Name),
```

Comment:
This can simply be `Name: e.accessPoint.Name`

Reply:
```suggestion
		Name:      aws.String(e.accessPoint.Name),
```